src/components/SubscriptionsPage.vue

```diff
@@ -15,7 +15,7 @@
             <div class="flex justify-center place-items-center">
                 <div class="w-full grid grid-cols-3">
                     <router-link :to="subscription.url" class="col-start-2 block flex text-center font-bold text-4xl">
-                        <img :src="subscription.avatar" class="rounded-full" width="48" height="48" />
+                        <img :src="subscription.avatar" class="rounded-full channel" width="48" height="48" />
```

Comment:
The channel class could just be replaced to `h-[fit-content]`

Alternatively, a class for channel could be created:
```
.channel {
    @apply rounded-full h-[fit-content];
}
```

These are just WindiCSS directives :)

Reply:
Nice :P I am a total vue / windicss noob so I tried to stick with base HTML+CSS. I honestly didn't think of just changing height from 48 there lol